src/sc/sidechainrpc.cpp

```diff
@@ -3,10 +3,18 @@
 #include <univalue.h>
 #include "primitives/transaction.h"
 #include <boost/foreach.hpp>
+#include <rpc/protocol.h>
+#include "utilmoneystr.h"
 
+#include <wallet/wallet.h>
 
 extern UniValue ValueFromAmount(const CAmount& amount);
 extern CAmount AmountFromValue(const UniValue& value);
+extern CWallet* pwalletMain;
+extern CFeeRate minRelayTxFee;
+extern std::string EncodeHexTx(const CTransaction& tx);
+extern UniValue signrawtransaction(const UniValue& params, bool fHelp);
+extern UniValue sendrawtransaction(const UniValue& params, bool fHelp);
```

Comment:
You should be able to change these externs with the following includes
#include <core_io.h>
#include <rpc/server.h>
#include <main.h>
#include <init.h>